typingdna_activelock/README.md

```diff
@@ -0,0 +1,54 @@
+
+
+# TypingDNA ActiveLock
+
+## Overview
+
+[TypingDNA ActiveLock][3] is a Continuous Endpoint Authentication app that helps prevent unauthorized access to your company computers. Once installed on a user's PC, it continuously verifies the user by the way they type. If an unauthorized typing pattern is detected, ActiveLock can instantly lock the computer and log the data to your desired logging platform (such as Datadog).
```

Comment:
Can you provide more insight on the value of the integration with Datadog? For example, "With this integration, you can xyz"

Reply:
Added more context.